packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "datacenter_backup_legacy_message_1": "Depuis le 25/02/2020, la gamme Veeam Backup a évolué.",
```

Comment:
please read date (25/02/2020) from the constant file. If the date is changed for some reason, we have to go through the translations again.